.circleci/config.yml

```diff
@@ -1,80 +1,75 @@
 version: 2
 
+## Build conditions
+# Build in any branch or tag
+build_always: &build_always
+  filters:
+    tags:
+      only: /^v.*/
+# Build only in master or in tags
+build_on_master: &build_on_master
+  filters:
+    tags:
+      only: /^v.*/
+    branches:
+      only: master
+# Build only in tags (release)
+build_on_tag: &build_on_tag
+  filters:
+    tags:
+      only: /^v.*/
+    branches:
+      ignore: /.*/
+
 workflows:
   version: 2
   kubeapps:
     jobs:
       - test_go:
-          filters:
-            tags:
-              only: /^v.*/
+          <<: *build_always
       - test_dashboard:
-          filters:
-            tags:
-              only: /^v.*/
+          <<: *build_always
       - build_chartrepo:
+          <<: *build_always
           requires:
             - test_go
-          filters:
-            tags:
-              only: /^v.*/
       - build_chartsvc:
+          <<: *build_always
           requires:
             - test_go
-          filters:
-            tags:
-              only: /^v.*/
       - build_chart_apprepository:
+          <<: *build_always
           requires:
             - test_go
-          filters:
-            tags:
-              only: /^v.*/
       - build_dashboard:
+          <<: *build_always
           requires:
             - test_dashboard
-          filters:
-            tags:
-              only: /^v.*/
       - build_tiller_proxy:
+          <<: *build_always
           requires:
             - test_go
-          filters:
-            tags:
-              only: /^v.*/
       - GKE_1_9:
+          <<: *build_on_master
```

Comment:
shouldn't we run integration tests on tagged releases too?

Reply:
yes, all the builds are done in tagged releases (I added some comments to the definitions)